hscommon/hsfiletypes/filetype.go

```diff
@@ -43,31 +43,28 @@ func determineTBLtype(data *[]byte) (FileType, error) {
 		return FileTypeTBLStringTable, nil
 	}
 
-	d := *data
-	if string(d[:4]) == "Woo!" {
+	if string((*data)[:4]) == "Woo!" {
 		return FileTypeTBLFontTable, nil
 	}
 
 	return FileTypeText, nil
 }
 
-func fileExtensionInfo() map[FileType]fileTypeInfoStruct {
-	return map[FileType]fileTypeInfoStruct{
```

Comment:
Make map's key the extension to prevent duplicate extensions in the list.

Reply:
look at abe7d28